src/main/external-resources/renderers/Samsung-9series.conf

```diff
@@ -30,11 +31,11 @@ LoadingPriority = 1
 
 TranscodeVideo = MPEGTS-H264-AC3
 TranscodeAudio = WAV
-#MaxVideoWidth = 3840
-#MaxVideoHeight = 2160
+MaxVideoWidth = 3840
+MaxVideoHeight = 2160
 SupportedVideoBitDepths = 8,10
 H264Level41Limited = true
-#H265Level41Limited = true
+H265Level41Limited = true
```

Comment:
OUPS..i forgot that it was here in the attempts of the on the way implementation 8/